google-cloud-core/src/main/java/com/google/cloud/testing/BaseEmulatorHelper.java

```diff
@@ -295,7 +295,7 @@ public Process getProcess() {
     private boolean isGcloudInstalled() {
       Map<String, String> env = System.getenv();
       for (String envName : env.keySet()) {
-        if ("PATH".equals(envName)) {
+        if ("PATH".equalsIgnoreCase(envName)) {
```

Comment:
I'm a little uneasy about this. In UNIX, these variables are case-sensitive. If `path` is set but `PATH` is not, we'll later crash because we'll try to invoke a command that's not in `PATH`.

Reply:
+1.